src/CLI/clitree/cli-xml/sonic_types.xml

```diff
@@ -514,28 +520,42 @@ limitations under the License.
     <PTYPE
         name="PTP_DELAY_MECHANISM"
         method="select"
-		pattern="e2e(E2E) p2p(P2P)"
+	pattern="e2e(E2E) p2p(P2P)"
         help=""
         />
     <!--=======================================================-->
     <PTYPE
         name="PTP_TRANSPORT"
         method="select"
-		pattern="l2(L2) ipv4(UDPv4) ipv6(UDPv6)"
+	pattern="l2(L2) ipv4(UDPv4) ipv6(UDPv6)"
         help=""
         />
     <!--=======================================================-->
     <PTYPE
         name="PTP_DOMAIN_PROFILE"
         method="select"
-		pattern="default(ieee1588) g8275.1(G.8275.x) g8275.2(G.8275.x)"
+	pattern="default(ieee1588) g8275.1(G.8275.x) g8275.2(G.8275.x)"
         help=""
         />
     <!--=======================================================-->
     <PTYPE
         name="PTP_PORT_MODE"
         method="select"
-		pattern="transparent(transparent) boundary(boundary) disable(disable)"
+	pattern="transparent(transparent) boundary(boundary) disable(disable)"
+        help=""
+        />
+    <!--=======================================================-->
+    <PTYPE
+        name="SFLOW_POLL_INTERVAL"
+        method="unsignedInteger"
+        pattern="0..300"
```

Comment:
You could do a regex here to limit it to 0,5-300

Reply:
This is caught by the sFlow YANG which has better error message than the xml file.